docs/source/library/simple_simulation.rst

```diff
@@ -0,0 +1,6 @@
+simple_simulation
+=================
+
+Function representing the simulation of a circuit.
```

Comment:
Nitpick: the function does not "represent" the simulation of a circuit. This is the function for _performing_ simulations.

Reply:
done 👍🏻